File: Hospital.Server/Entities/Response/TimezoneResponse.cs
namespace Hospital.Server.Entities.Response
{
    /// <summary>
    /// Defines the <see cref="TimezoneResponse" />
    /// </summary>
    public class TimezoneResponse
    {
        /// <summary>
        /// Gets or sets the Id
        /// </summary>
        public long Id { get; set; }

        /// <summary>
        /// Gets or sets the IANA timezone identifier
        /// </summary>
        public string IanaId { get; set; } = string.Empty;

        /// <summary>
        /// Gets or sets the display name
        /// </summary>
        public string DisplayName { get; set; } = string.Empty;

        /// <summary>
        /// Gets or sets the UTC offset string
        /// </summary>
        public string UtcOffset { get; set; } = string.Empty;

        /// <summary>
        /// Gets or sets the State
        /// </summary>
        public int State { get; set; }
    }
}

File: hospital.client/src/types/TimezoneResponse.ts
export interface TimezoneResponse {
  id: number;
  ianaId: string;
  displayName: string;
  utcOffset: string;
  state: number;
}

export interface TimezoneRequest {
  id?: number | null;
  ianaId?: string | null;
  displayName?: string | null;
  utcOffset?: string | null;
  state?: number | null;
  createdBy?: number | null;
  updatedBy?: number | null;
}
